CONTRIBUTING.md

```diff
@@ -71,8 +71,8 @@ The core team is monitoring for Pull Requests. We will review your Pull Request
 ### How to increase the chance of being accepted?
 
 CI runs a series of checks automatically when a Pull Request is opened. If you're not
-sure if you changes will pass, you can always open a Pull Request and the GitHub UI will display a summary of
-the results. If one of them fails check out the section [Checks and how to fix them](#checksfix).
+sure if your changes will pass, you can always open a Pull Request and the GitHub UI will display a summary of
+the results. If one of them fails, check out the section [Checks and how to fix them](#checksfix).
```

Comment:
Nice, what do you think of - ‘If any of them fail,’ cc @mbrookes

Reply:
Both seem okay to me. I'm not a native speaker though :)

Reply:
@joshwooding I can't seem to agree with myself 😄. Is there a higher power we can appeal to?

Reply:
@mbrookes You were my higher power 😅If we cant decide/it’s not clear, not changing it seems the best option 😄

Reply:
> You were my higher power

Also mine 🤓

Reply:
My dilemma was with whether "any" is singular or plural in this instance. Since more than one could fail, that suggests it's plural, in which case your subject/verb agreement in the suggestion is correct.